pkg/webui/console/components/pubsub-form/mapping.js

```diff
@@ -59,19 +55,45 @@ export const mapNatsServerUrlToFormValue = function(server_url) {
   }
 }
 
+export const mapMqttFormValues = function(mqtt) {
+  if (!mqtt.tls_ca) {
+    mqtt.tls_ca = ''
+  }
+  if (!mqtt.tls_client_cert) {
+    mqtt.tls_client_cert = ''
+  }
+  if (!mqtt.tls_client_key) {
+    mqtt.tls_client_key = ''
+  }
+  if (!mqtt.username) {
+    mqtt.username = ''
+  }
+  if (!mqtt.password) {
+    mqtt.password = ''
+  }
+  mqtt.use_credentials = Boolean(mqtt.username || mqtt.password)
+  return mqtt
+}
```

Comment:
Let's merge with the blank object instead:

```javascript
return merge({ use_credentials: Boolean(mqtt.username || mqtt.password) }, mqtt, mqttBlankValues)
```

(Also nit: `use_credentials` should really be `_use_credentials`, to be in line with how we name meta fields that are not part of the registry, seems like this slipped through a past PR review).